MAINTAINERS.yml

```
@@ -5179,6 +5179,20 @@ Utilities:
     - libraries.ring_buffer
     - libraries.linear_range
 
+Uuid:
+  status: maintained
+  maintainers:
+    - sorru94
+  files:
+    - lib/uuid/
+    - include/zephyr/sys/uuid.h
+    - tests/lib/uuid/
+    - samples/subsys/uuid/
+  labels:
+    - "area: Uuid"
+  tests:
+    - libraries.uuid
```

Use capital letters for an abbreviations (will need to move below USB-C for alphabetical order)

```suggestion
UUID:
  status: maintained
  maintainers:
    - sorru94
  files:
    - lib/uuid/
    - include/zephyr/sys/uuid.h
    - tests/lib/uuid/
    - samples/subsys/uuid/
  labels:
    - "area: UUID"
  tests:
    - libraries.uuid
```